vk-video/src/parser/reference_manager.rs

```diff
@@ -36,6 +39,9 @@ pub enum ReferenceManagementError {
 
     #[error("pic_order_cnt_lsb is not present in a slice header, but is required for decoding")]
```

Comment:
remove this error since you added a generic 'bad bytestrram' one dummy.

also remove the one about not supporting ref list modification.